lib/restful_resource/base.rb

```diff
@@ -58,7 +58,7 @@ def self.get(**params)
     def self.delete(id, **params)
       params_without_options, options = format_params(params)
       response = http.delete(member_url(id, params_without_options), **options)
-      RestfulResource::OpenObject.new(parse_json(response.body))
+      new(parse_json(response.body))
     end
```

Comment:
Given that Base inherits from OpenObject I guess this should not cause any issues in consumers (??)

Reply:
Exactly